pkg/cli/admin/prune/renderedmachineconfigs/renderedmachineconfigs.go

```diff
@@ -0,0 +1,426 @@
+package renderedmachineconfigs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"sort"
+	"time"
+	"unicode"
+
+	mcfgclientset "github.com/openshift/client-go/machineconfiguration/clientset/versioned"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/client-go/kubernetes"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+var (
+	pruneMCLong = templates.LongDesc(`
+		# Prune rendered machine configs for an OCP v4 cluster.
```

Comment:
nit: Could you please align those raw strings with an each new line between examples?